languages/common-lisp/reference/exercise-concepts/word-count.md

```diff
@@ -0,0 +1,39 @@
+# General Concepts of Word Count
+- Variables: Local variables with `let` or `let*` are commonplace
+- String: Strings need to be split into a list of words. Cleaning is also done
+  with `string-trim` and `string-downcase`
+- Functions: Helper functions are commonly defined
+- Sameness: Common lisp has a _lot_ of equality operators. Most useful here are:
+  `equal` and `string=`
+- Comments: We can hope for well commented code (this includes the use of
+  docstrings!)
+- Nested Functions: Often appear as string cleaners or serve other
+  functions. Defined with `flet` or `labels`
+- Higher Order Functions: The use of various functions like `remove-if` and
+  `map` is common
+- Anonymous Functions: Some students may pass a `lambda` to functions like `map`
+- Lists: This exercise requires a result given as an `alist`
+- Boolean Logic: Compositions of `and` and `or`. Helpfully short-circuiting
+
+# String -> Words
+## Approach: Library Import
+- Regular Expressions: Needed if the student is using the `cl-ppcre`
+
+## Approach: Home Rolled
+- Enumeration: Common Lisp's `loop` macro makes it possible to loop through the
```

Comment:
What about `do`/`do*`? Perhaps that is not an issue of concepts but for the analyzer/recognizer to know that `do` /or/ `loop` can be used.

Thoughts?

Reply:
That's a good point! I didn't see too many solutions using `do`, but I should definitely include it here. I'll make that change tomorrow hopefully